scripts/backport.py

```diff
@@ -369,7 +378,7 @@ def __init__(self, pygithub_pr_, issue_number_):
         head=backport_branch,
         base=backport_target,
     )
-    backport_pr.add_to_labels("pr-backport")
+    backport_pr.add_to_labels("pr-is-backport")
```

Comment:
Our labels are all over the place.
```suggestion
    backport_pr.add_to_labels("auto-backport")
```

Reply:
Do we have some general idea of how they should look? The reason I'm wording it this way is that we need labels for both "please do backport this PR/issue" and "this PR is an automatically created backport". `auto-backport` is amgibuous between these two. Here's what I have now: https://github.com/timescale/timescaledb/labels?q=backport+

Reply:
And we probably also want the "please-backport" label to apply on PRs as well, because sometimes we don't have a corresponding bug issue, like here: https://github.com/timescale/timescaledb/pull/5259

Reply:
I have no strong opinion on what labels to use, but it would be good if we could be more planned approach and not add them ad-hoc. We are going to have to manage these labels through the normal GitHub interface as well, so good and clear naming is part of the problem.

If you could write down a concrete proposal with the labels that we can have a quick chat around, it might be sufficient.

Reply:
Sure, I'll do that. I'll keep them as is for now, because we have to test it anyway, and after the discussion will rename them. I added an item for this to the epic: https://github.com/timescale/eng-database/issues/347